elements/media-quote/src/media-quote.js

```diff
@@ -23,6 +24,140 @@ class MediaQuote extends LitElement {
    */
   constructor() {
     super();
+    
+    // Media Item (Img)
+    this.mediaSrc = '';
+    this.mediaAlt = '';
+    this.caption = '';
+    
+    // Citation
+    this.author = '';
+    this.authorProfession = '';
+
+    // Color
+    this.dataPrimary;
+  }
+
+  static get styles() {
+    return [
+      super.styles,
+        css`
+          /* https://oer.hax.psu.edu/bto108/sites/haxcellence/documentation/ddd */
+          .media-quote-container {
+            font-family: var(--ddd-font-primary);
+            display: flex;
+            justify-content: center;
+          }
+          
+          .text-overlay {
+            z-index: 2;
+          }
+
+          .quote {
+            display: inline-block;
+            padding: var(--ddd-spacing-0) var(--ddd-spacing-3);
+            font-style: italic;
+            width: 30%;
+            box-decoration-break: clone;
+            font-size: var(--ddd-font-size-ms);
+            position: absolute;
+            top: 20%;
+            left: 15%
+          }
+
+          .content {
+            box-decoration-break: clone;
+          }
+
+          .content, .citation {
+            background-color: var(--ddd-theme-default-beaverBlue);
+            padding: var(--ddd-spacing-0) var(--ddd-spacing-2);
+          }
+
+          .citation {
+            display: inline-block;
+            font-style: italic;
+            font-size: 0;
+          }
+            .author {
+              font-weight: bold; /* var(--ddd-font-primary-bold) */
```

Comment:
use our bold system. I believe its actually `--ddd-font-weight-bold` If you go into the DDDStyles.js file this has the latest vars which are currently ahead of what's published on the DDD site you guys used in class.